tgui/packages/tgui/interfaces/RobotSelfDiagnosis.js

```diff
@@ -0,0 +1,87 @@
+import { useBackend } from "../backend";
+import { NoticeBox, Flex, LabeledList, Section } from "../components";
+import { Window } from "../layouts";
+import { capitalize } from 'common/string';
+
+export const RobotSelfDiagnosis = (props, context) => {
+  const { data } = useBackend(context);
+  const { component_data } = data;
+  const getDamageColor = (damage, maxDamage) => {
+    let damageRatio = damage / maxDamage;
+    if (damageRatio <= 0.20) {
+      return "good";
+    }
+    else if (damageRatio <= 0.50) {
+      return "average";
+    }
+    else {
+      return "bad";
+    }
+  };
+  const getStatusColor = status => {
+    return status ? "good" : "bad";
+  };
+  return (
+    <Window>
+      <Window.Content scrollable>
+        {component_data.map((entry, i) => (
+          <Section
+            key={i}
+            open
+            title={capitalize(entry.name)}>
+            {entry.installed <= 0 ? (
+              <NoticeBox
+                m={-0.5}
+                height={3.5}
+                color="red"
+                style={{
+                  "font-style": "normal",
+                }}>
+                <Flex height="100%">
+                  <Flex.Item
+                    grow="1"
+                    textAlign="center"
+                    align="center"
+                    color="#e8e8e8">
```

Comment:
Is it needed to specify the colour here? It doesn't seem to make too much difference if it's just on the default setting.
Default:
![normal](https://user-images.githubusercontent.com/57483089/125328043-c0135480-e33b-11eb-8e36-5c8494ada17d.PNG)
Modified:
![modified](https://user-images.githubusercontent.com/57483089/125328065-c73a6280-e33b-11eb-8bea-a6d237190c1d.PNG)

Reply:
I thought the white was a smidge too bright, especially when using `color="red"` for the warning color (discussed in your other review).